forms.rst

```diff
@@ -702,6 +702,16 @@ the choice is ultimately up to you.
 
         $form->get('dueDate')->setData(new \DateTime());
 
+Rendering the Same Form Multiple Times on the Same Page
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In some situations you might find yourself in need of having the same form on
+the same page multiple times. By default, a form name is based off the class of
+the form type. To differentiate between the forms, you can create a form using
+:method:`FormFactoryInterface::createNamed() <Symfony\\Component\\Form\\FormFactoryInterface>`.
+Doing so, will change the html name of the form and Symfony will be able to see
+which of the forms was submitted, and handle it accordingly.
```

Comment:
I understand the need to document this method, but I propose to just say something like:
```rst
A form name is generated from the type class name, if you want to set it explicitly use ..., you can event suppress it by explicitly set it to an empty string.
```